go/private/coverage_shim

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
Explain why this doesn't have a go extension (or add one)

Reply:
I wasn't sure if anyone runs gazelle manually on the repo and didn't want it creating a dummy target